resources/assets/js/features/search/components/Result/Result.js

```diff
@@ -0,0 +1,193 @@
+import React from 'react';
+import Filter from './Filter';
+import SearchForm from './SearchForm';
+import TripList from './TripList';
+import SortPanel from './SortPanel';
+import Pagination from './Pagination';
+import { connect } from 'react-redux';
+import { search, encodeCoord, getDataFromQuery, setUrl, setFilter} from '../../services/SearchService';
+import { searchResult } from '../../actions';
+import { bindActionCreators } from 'redux';
+import { withRouter, browserHistory } from 'react-router';
+import '../../styles/search-result.scss';
+
+class Result extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            collection: [],
+            meta: {
+                totalSize: 0,
+                priceRange: [0,0]
+            },
+            preloader: true,
+            sort: props.location.query.sort || 'price',
```

Comment:
avoid using global props location on this level, better pass needed data into component